lib/bone.js

```diff
@@ -760,14 +759,14 @@ class Bone {
 
   static async bulkCreate(records) {
     const { driver, table, attributes, primaryKey, primaryColumn } = this;
-    const _setDefaultValue = (record) => {
+    const _setDefaultValue = record => {
       for (const name in attributes) {
         const value = record[name];
         // set defaultValues
         const { defaultValue } = attributes[name];
         if (value === undefined && defaultValue != null) record[name] = defaultValue;
       }
-    }
+    };
```

Comment:
regular function should be preferred if arrow function is not necessary, since there's no `this` to bind